packages/web/src/client/components/Site/SiteLayout.tsx

```diff
@@ -34,6 +36,44 @@ export const Layout = ({
 
   const [showSplash, setShowSplash] = useState(true)
 
+  // :: ------------------------------------------------------
+  let counter = 0
+
+  setInterval(() => {
```

Comment:
Here's an article that will help you understand how to use 'setInterval' in react - https://sebhastian.com/setinterval-react/